packages/ui/src/ui/pages/operations/OperationsList/OperationsListToolbar/OperationsListToolbar.js

```diff
@@ -222,7 +222,7 @@ class OperationsListToolbar extends React.PureComponent {
 
     renderWarning_uiissue_2838() {
         return (
-            <div className={block('ytfront-2383')}>
+            <div className={block('ytfront-2838')}>
```

Comment:
I changed css class here because the issue numbers are different. I guess that ytfront-2838 is the right one and ytfront-2383 is just a typo?